contracts/src/traits/errors/psp22.rs

```diff
@@ -105,6 +105,8 @@ pub enum PSP22TokenTimelockError {
     NoTokensToRelease,
     /// Returned if the timestamp provided is before the current time
     ReleaseTimeIsBeforeCurrentTime,
+    /// Returned if either the token or the beneficiary is `None`
+    ZeroAddress,
```

Comment:
Let's call it another name, e.g. AccountDoesNotExist

Reply:
because this way, user can use zero address, so this naming of error looks strange